frontend/src/modules/dataSources/components/DataSourceManager/DataSourceManager.jsx

```diff
@@ -138,6 +139,11 @@ class DataSourceManagerComponent extends React.Component {
   };
 
   selectDataSource = (source) => {
+    posthogHelper.captureEvent('choose_datasource', {
```

Comment:
Shall we match the properties (source) ?

`    posthog.capture('choose_datasource', { dataSource: source.kind, category, appId: this.state.appId });
`
Copied from the current cloud code